cmd/ctl/cmd.go

```diff
@@ -34,8 +38,15 @@ cert-manager-ctl is a CLI tool manage and configure cert-manager resources for K
 		Run: runHelp,
 	}
 
+	kubeConfigFlags := genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag()
```

Comment:
Should we be registering the 'deprecated password flag'? Seems like we don't really need to if it is deprecated upstream?

Reply:
whoops